java/test7/A.java

```diff
@@ -30,6 +30,7 @@ public void run() throws DDlogException {
         DDlogRecord rec = c.value();
         System.out.println(rec.toString());
         result.dispose();
+        this.api.dumpIndex("AII", i -> System.out.println(i));
```

Comment:
This test doesn't seem to have reference output.

Also, this test is not currently part of the test suite.  Can you add it to `test.sh`, by analogy with `java1`,...,`java4` and do `.gitlab-ci.yml`?